plugins/outputs/cloudwatch/cloudwatch_test.go

```diff
@@ -375,40 +373,50 @@ func TestWriteError(t *testing.T) {
 	svc.On("PutMetricData", mock.Anything).Return(
 		&res,
 		serverInternalErr)
-	cloudWatchOutput := newCloudWatchClient(svc)
-	cloudWatchOutput.publisher, _ = publisher.NewPublisher(publisher.NewNonBlockingFifoQueue(10), 10, 2*time.Second, cloudWatchOutput.WriteToCloudWatch)
-	metrics := make([]telegraf.Metric, 0, 2)
-	measurement := "Test_namespace"
-	fields := map[string]interface{}{
```

Comment:
What are the reasons to delete the tags and dimensions though?

Reply:
see the other comment and response on line 350.